Documentation/games/SkyClash.md

```diff
@@ -1,7 +1,11 @@
+### SkyClash has been removed from the Hypixel Network. Read the forum post [here](https://hypixel.net/threads/clarifications-about-skyclash.2031767/).  The API will still stay functional.
```

Comment:
Should I also keep this? Or make it smaller / shorter

Reply:
Maybe change it to `### SkyClash` and have a subheader saying `#### SkyClash has been removed` with the rest of the text?